### PR TITLE
gpio: Python: improve the gpio sample

### PR DESCRIPTION
**Pull request by AllanKamimura**
This MR is re-implements the changes discussed in https://github.com/toradex/torizon-samples/pull/26.

But with better file changes tracking. With this, this MR should be merged in place of https://github.com/toradex/torizon-samples/pull/26

### CODE REVIEW

File: gpio/python/gpioPython/docker-compose.yml [hunk 1]
```diff
@@ -8,13 +8,9 @@ services:
     ports:
       - 6502:6502
       - 6512:6512
-    devices:
```
Comment (andreravila) [inline gpio/python/gpioPython/docker-compose.yml:11]: I think it is better to remove the changes on `docker-compose.yml` from this commit, and make them in the other commit

File: gpio/python/gpioPython/.vscode/settings.json [hunk 2]
```diff
@@ -1,18 +1,22 @@
 {
-  "torizon_psswd": "",
-  "torizon_login": "",
-  "torizon_ip": "",
-  "host_ip": "",
+  "torizon_psswd": "1",
+  "torizon_login": "torizon",
+  "torizon_ip": "10.22.1.63",
```
Comment (andreravila) [inline gpio/python/gpioPython/.vscode/settings.json:4]: "Remove Default" on the device before commiting, so it doesn't add your devices configuration

File: gpio/python/gpioPython/docker-compose.yml (hunk 1 above)
Comment (andreriesco) [inline gpio/python/gpioPython/docker-compose.yml:11]: I think it is better to remove the changes on `docker-compose.yml` from this commit, and make them in the other commit

File: gpio/python/gpioPython/.vscode/settings.json [hunk 3]
```diff
@@ -1,18 +1,22 @@
 {
-  "torizon_psswd": "",
-  "torizon_login": "",
-  "torizon_ip": "",
-  "host_ip": "",
+  "torizon_psswd": "1",
+  "torizon_login": "torizon",
```
Comment (andreriesco) [inline gpio/python/gpioPython/.vscode/settings.json]: "Remove Default" on the device before commiting, so it doesn't add your device's configuration

**Comment by AllanKamimura** [comment]
Hi @andreriesco 
I applied the commit edits you mentioned
I had some trouble with the rebase :upside_down_face: , but I think it is ok now

File: gpio/python/gpioPython/torizonPackages.json [hunk 4]
```diff
@@ -1,8 +0,0 @@
-{
-    "deps": [
-        "python3-libgpiod"
```
Comment (andreriesco) [inline gpio/python/gpioPython/torizonPackages.json:3]: Here remove just the `python3-libgpiod` lines

**Comment by otaviocp** [comment]
@andreriesco Can you check if this branch is ready to merge?

**Comment by andreriesco** [comment]
Almost ready, missing just [this fix](https://github.com/toradex/torizon-samples/pull/29#discussion_r1681084733)

**Comment by otaviocp** [comment]
The [Python Console template](https://github.com/toradex/vscode-torizon-templates/tree/dev/python3Console) does not have the 'torizonPackages.json' file. Should I just remove it?

File: gpio/python/gpioPython/torizonPackages.json (hunk 4 above)
Comment (otaviocp) [reply gpio/python/gpioPython/torizonPackages.json:3]: Done. Thanks!

**Comment by andreriesco** [comment]
Now LGTM

**Pull request merged**